src/composables/useUsername.ts

```diff
@@ -11,6 +11,21 @@ export function useUsername() {
     ens: ''
   });
 
+  const fullName = computed(() => {
+    if (
+      web3Account.value &&
+      address.value.toLowerCase() === web3Account.value.toLowerCase()
+    ) {
+      return 'You';
+    }
+    if (profile.value?.name) {
+      return profile.value.name;
+    } else if (profile.value?.ens) {
+      return profile.value.ens;
+    }
+    return address.value;
+  });
```

Comment:
There's quite a bit of duplicated code here. I think `username` can simply use `fullName`. Analogous to `username` I'd also name it `fullUsername`.